src/pages/nft/class/_classId/index.vue

```diff
@@ -345,7 +355,12 @@ export default {
       this.updateNFTHistory();
       this.lazyFetchLIKEPrice();
       this.fetchUserCollectedCount();
-      const blockingPromises = [this.fetchISCNMetadata()];
+      const blockingPromises = [
+        this.fetchISCNMetadata(),
+        this.fetchNFTListByAddress(this.iscnOwner),
+        this.fetchNFTListByAddress(this.getAddress),
+        this.fetchNFTDisplayStateListByAddress(this.iscnOwner),
```

Comment:
Must these be blocking, or can we unblock, and show recommendation section after these are loaded?